server/src/test/java/org/elasticsearch/index/mapper/vectors/DenseVectorFieldMapperTests.java

```diff
@@ -1022,6 +1022,60 @@ public void testInvalidRescoreVector() {
         }
     }
 
+    public void testDefaultOversampleValue() throws IOException {
```

Comment:
This is good, but please add a yaml test that verifies this default.

Reply:
yaml tests will require a test node feature so you can skip them in earlier versions where the default was not set.